kie-server-parent/kie-server-controller/kie-server-controller-rest/src/main/java/org/kie/server/controller/rest/RestKieServerControllerImpl.java

```diff
@@ -64,13 +70,15 @@ public Response connectKieServer( @Context HttpHeaders headers, @PathParam("id")
 
         logger.debug("Returning response for connect of server '{}': {}", id, response);
         return createCorrectVariant(response, headers, Response.Status.CREATED);
-
     }
 
+    @ApiOperation(value="Remove a Kie Server instance from Controller")
+    @ApiResponses(value = { @ApiResponse(code = 500, message = "Unexpected error") })
     @DELETE
     @Path("server/{id}")
+    @Consumes({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
+    @Produces({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
```

Comment:
I don't think this produces any content, most likely returns 204

Reply:
That's true, I removed this one and reviewed some other endpoints where no content was generated.